custom-exporter/exporter.py

```diff
@@ -140,13 +240,20 @@ def get_phase(self, address: str) -> str:
         except ContractLogicError as e:
             logger.error(f"Error calling get_phase: {e}")
             raise
+
     def update_metrics(self):
         try:
             node_info = self.get_node()
             group_index, _ = self.get_belonging_group()
-            
+            self.current_node_status = node_info[3]
+            if(self.current_node_status and self.prev_node_status):
+                self.accumulate_uptime(self.config['node_address'])
+            else:
+                self.update_uptime(self.config['node_address'])
+
+            self.prev_node_status = self.current_node_status
```

Comment:
I think we can implement the existing logic only with `self.current_node_status`, this `self.prev_node_status` makes me confused.